Test/ServerTelemetryChannel.Test/Shared.Tests/Implementation/TransmissionStorageTest.cs

```diff
@@ -529,7 +529,7 @@ public void LoadsTransmissionFromTheOldestTransmissionFileInTheFolder()
 
                 var oldestAddress = new Uri("http://oldest");
                 StubPlatformFile oldestFile = CreateTransmissionFile(oldestAddress);
-                oldestFile.OnGetDateCreated = () => DateTimeOffset.MinValue;
+                oldestFile.OnGetDateCreated = () => DateTimeOffset.Now.AddDays(-1);
```

Comment:
Why this change? Can you please leave a comment in code for the magic number of days